index.js

```diff
@@ -53,6 +53,16 @@ function typedArraySupport () {
   }
 }
 
+Object.defineProperty(Buffer.prototype, 'offset', {
+  enumerable: true,
```

Comment:
This should probably be `enumerable: false`, since we don't want these properties to show up in `Object.keys()`. Here's what Node.js does:

```js
Object.keys(Buffer.from([1, 2,3]))
// [ '0', '1', '2' ]
```